dcpy/lifecycle/package/validate.py

```diff
@@ -126,14 +126,17 @@ def _is_geom_poly(s):
 
 
 def validate_df(
-    df: pd.DataFrame, columns: list[dataset_md.DatasetColumn]
+    df: pd.DataFrame,
+    columns: list[dataset_md.DatasetColumn],
+    *,
+    ignore_columns: list[str] | None = None,
 ) -> list[ValidationError]:
     """Validate a dataframe against a metadata file."""
     df_stringified_nulls = df.fillna("")
 
     errors = []
 
-    column_names = {c.name for c in columns}
+    column_names = {c.name or "" for c in columns}
```

Comment:
There a reason we don't want `{c.name for c in columns if c}` here?

Reply:
Yeah, I've added a comment about that in the last commit. It was mostly about appeasing the type-checker around c.name's nullability. But it points to the need to change the Column model.